### ISSUE TITLE
RedisModule makes Jest testing hang

### ISSUE THREAD

**Issue by germansokolov13** [kyknow/nestjs-redis#37]
# Description
Nest.js project by default comes with a nice integration with Jest testing framework. Injecting `RedisModule` into testing module causes tests command to hang after execution and not exit properly.

I already experienced such problems with testing frameworks connecting to databases. It might be resolved by somehow disconnecting from Redis once all tests have passed. The docs do not say how to disconnect.

Reasons to import it into tests:
- I may create integration tests that actually use Redis
- I may create a client to only mock its methods after that
- I have many tests that just import the entire AppModule as is along with RedisModule. Then I test services that may not even actually use the module.
# Versions
- nestjs: 6.11.7
- jest: 24.9.0
- nestjs-redis: 1.2.5
# Steps to reproduce
Use `nest new` command from `@nestjs/cli` to create project. At this point `npm run test` correctlly exits after execution. Now install `nestjs-redis` and inject it inside testing module.

That's `app.controller.spec.ts` file
```JS
const app: TestingModule = await Test.createTestingModule({
  imports: [
    RedisModule.register({
      host: 'localhost',
      port: 6379,
    })
  ],
  controllers: [AppController],
  providers: [AppService],
}).compile();
```
Now run `npm run test`. The test is green but it does not exit, the command hangs with this output:

> Jest did not exit one second after the test run has completed.
> 
> This usually means that there are asynchronous operations that weren't stopped in your tests. Consider running Jest with `--detectOpenHandles` to troubleshoot this issue.
`detectOpenHandles` flag does not help.

**Comment by germansokolov13** [comment]
Actually there already is a PR to solve it:
https://github.com/kyknow/nestjs-redis/pull/24
It existed before I made an issue but it's not merged...

**Comment by a-pavlenko** [comment]
Issue was closed, but #24 has not been released yet, any idea when the new version could be published?
